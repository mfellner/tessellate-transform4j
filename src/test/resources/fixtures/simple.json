{
  "class": "div",
  "id": "test",
  "ignore": 42,
  "children": [
    {
      "class": "h1",
      "id": "heading",
      "contents": ["Hello, world!"]
    }
  ]
}
