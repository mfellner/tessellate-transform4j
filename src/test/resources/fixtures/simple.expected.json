{
  "type": "test.Wrapper",
  "props": null,
  "children": [
    {
      "type": "div",
      "props": {
        "id": "test"
      },
      "children": [
        {
          "type": "test.Heading",
          "props": {
            "id": "heading"
          },
          "children": [
            "Hello, world!"
          ]
        }
      ]
    }
  ]
}
